ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -1563,20 +1575,12 @@ generatePrerequisites ghcFlavor = do
         =<< readFile' "./mk/get-win32-tarballs.sh"
     )
 
-  -- When there is a new GHC release it takes time for package bounds
-  -- to get updated.
-#if __GLASGOW_HASKELL__ == 912
-  let hadrianExtraCabalFlags = "--allow-newer "
-#else
-  let hadrianExtraCabalFlags = ""
-#endif
-
   system_ "bash -c ./boot"
   system_ "bash -c \"./configure --enable-tarballs-autodownload\""
   withCurrentDirectory "hadrian" $ do
-    system_ $ "cabal build " ++ hadrianExtraCabalFlags ++ "exe:hadrian --ghc-options=-j"
+    system_ $ "cabal build exe:hadrian --ghc-options=-j"
```

Comment:
The warning suggests we drop this dollar, up to you

Reply:
i pushed a new commit for it.